screens/Records/AddEditRecord.js

```diff
@@ -0,0 +1,158 @@
+import React,{useState} from 'react'
+import {View,SafeAreaView,ScrollView,Text,TouchableOpacity,Button,Modal} from 'react-native'
+import tw from 'twrnc'
+import {TextInput} from 'react-native-paper'
+import DropDown from 'react-native-paper-dropdown'
+import DatePicker from 'react-native-date-picker'
+import DatePick from './datepick'
+import StartTime from './startTime'
+import EndTime from './endTime'
+import { MyButton } from '../Tractors/AddEditTractor'
+
+const driverlist = [
```

Comment:
Why do we need this default value? Don't we take it form storage?

Reply:
yes Anna,I have know completed UI only.If the UI is Okay then we can add storage values.